components/gitpod-db/src/typeorm/migration/1669892320740-PersonalAccessTokenFixExpirationTimeAutoUpdate.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+const TABLE_NAME = "d_b_personal_access_token";
+
+export class PersonalAccessTokenFixExpirationTimeAutoUpdate1669892320740 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE ${TABLE_NAME} CHANGE expirationTime expirationTime timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6)`,
```

Comment:
I was about to ask the question, but the ✅ came in and I lost track :-( 

☝🏻 there one  `expirationTime` too much in the statement.

Reply:
The change statement takes `from` and `to` arguments. Here we're changing the same one so both are set to the same column name
https://www.tutorialspoint.com/how-do-i-remove-on-update-current-timestamp-from-an-existing-column-in-mysql